aten/src/ATen/native/sparse/SparseUtils.h

```diff
@@ -108,16 +108,9 @@ inline LongTensor _newFlattenedIndices(const SparseTensor& self, bool forceClone
 // TODO: Expose this for real in ATen, some day?
 // NB: Doesn't preserve data.
 inline Tensor _new_values_with_size_of(const Tensor& values, int64_t nnz) {
-  if (values.numel() == 0) { // values tensor uninitialized
```

Comment:
I believe this should be removed after @yf225 's patch.

Reply:
Yes I have a PR coming up that will also make the same change in this function. Feel free to fix it in this PR first :)